README.md

```diff
@@ -9,6 +9,33 @@ data, collected data, and publications related to the Task 2 "Fuel Cycle modelin
 The documentation for this work can be found in the docs folder as well as this 
 readme. 
```

Comment:
Lines 2 and 4 in source and your branch are kind of redundant. Consider merging the two or deleting one.